lang/po/new/hu.po

```diff
@@ -2,1771 +2,1767 @@
 #: ultralcd.cpp:2885
 msgid "Measured skew"
-msgstr ""
+msgstr "Mert merol. hiba"
 
 # MSG_MMU_FAILS c=15
 #: messages.c:69
 msgid "MMU fails"
-msgstr ""
+msgstr "MMU hibak"
 
 # MSG_MMU_LOAD_FAILED c=20
 #: mmu.cpp:1587
 msgid "MMU load failed"
-msgstr ""
+msgstr "MMU betolt.hiba"
 
 # MSG_MMU_LOAD_FAILS c=15
 #: messages.c:70
 msgid "MMU load fails"
-msgstr ""
+msgstr "MMU bet. hibak"
 
 # MSG_MMU_OK_RESUMING c=20 r=4
 #: mmu.cpp:776
 msgid "MMU OK. Resuming..."
-msgstr ""
+msgstr "MMU OK. Folytatom..."
 
 # MSG_MODE c=6
 #: messages.c:103
 msgid "Mode"
-msgstr ""
+msgstr "Mod"
 
 # MSG_MK3_FIRMWARE_ON_MK3S c=20 r=4
 #: Marlin_main.cpp:882
 msgid "MK3 firmware detected on MK3S printer"
-msgstr ""
+msgstr "MK3 firmver telepitve MK3S nyomtatora"
 
 # MSG_NORMAL c=7
 #: messages.c:107
 msgid "Normal"
-msgstr ""
+msgstr "Normal"
 
 # MSG_SILENT c=7
 #: messages.c:106
 msgid "Silent"
-msgstr ""
+msgstr "Halk"
 
 # MSG_MMU_USER_ATTENTION c=20 r=3
 #: mmu.cpp:722
 msgid "MMU needs user attention."
-msgstr ""
+msgstr "Az MMU felhasznaloi figyelmet igenyel."
 
 # MSG_MMU_POWER_FAILS c=15
 #: ultralcd.cpp:1517
 msgid "MMU power fails"
-msgstr ""
+msgstr "MMU tap hibak"
 
 # MSG_STEALTH c=7
 #: messages.c:108
 msgid "Stealth"
-msgstr ""
+msgstr "Halk"
 
 # MSG_AUTO_POWER c=10
 #: messages.c:105
 msgid "Auto power"
-msgstr ""
+msgstr "Auto ero"
 
 # MSG_HIGH_POWER c=10
 #: messages.c:104
 msgid "High power"
-msgstr ""
+msgstr "Magas ero"
 
 # MSG_MMU_CONNECTED c=18
 #: ultralcd.cpp:1997
 msgid "MMU2 connected"
-msgstr ""
+msgstr "MMU2 csatlakozott"
 
 # MSG_SELFTEST_MOTOR c=18
 #: messages.c:96
 msgid "Motor"
-msgstr ""
+msgstr "Motor"
 
 # MSG_MOVE_AXIS c=18
 #: ultralcd.cpp:5677
 msgid "Move axis"
-msgstr ""
+msgstr "Tengely mozgatasa"
 
 # MSG_MOVE_X c=18
 #: ultralcd.cpp:4233
 msgid "Move X"
-msgstr ""
+msgstr "X mozgatasa"
 
 # MSG_MOVE_Y c=18
 #: ultralcd.cpp:4234
 msgid "Move Y"
-msgstr ""
+msgstr "Y mozgatasa"
 
 # MSG_MOVE_Z c=18
 #: ultralcd.cpp:4235
 msgid "Move Z"
-msgstr ""
+msgstr "Z mozgatasa"
 
 # MSG_NO_MOVE c=20
 #: Marlin_main.cpp:5852
 msgid "No move."
-msgstr ""
+msgstr "Nincs mozgas."
 
 # MSG_NO_CARD c=18
 #: ultralcd.cpp:6697
 msgid "No SD card"
-msgstr ""
+msgstr "Nincs SD kartya"
 
 # MSG_NA c=3
 #: messages.c:127
 msgid "N/A"
-msgstr ""
+msgstr "N/A"
 
 # MSG_NO c=4
 #: messages.c:71
 msgid "No"
-msgstr ""
+msgstr "Nem"
 
 # MSG_SELFTEST_NOTCONNECTED c=20
 #: ultralcd.cpp:8128
 msgid "Not connected"
-msgstr ""
+msgstr "Nincs csatlakoztatva"
 
 # MSG_NEW_FIRMWARE_AVAILABLE c=20 r=2
 #: util.cpp:195
 msgid "New firmware version available:"
-msgstr ""
+msgstr "Uj firmver verzio erheto el:"
 
 # MSG_SELFTEST_FAN_NO c=19
 #: messages.c:92
 msgid "Not spinning"
-msgstr ""
+msgstr "Nem forog"
 
 # MSG_WIZARD_V2_CAL c=20 r=8
 #: ultralcd.cpp:4838
 msgid "Now I will calibrate distance between tip of the nozzle and heatbed surface."
-msgstr ""
+msgstr "Most beallítjuk a fuvoka hegye es a targyasztal felulete kozotti tavolsagot."

```

Comment:
The `lang-import.sh hu` fails as `beallítjuk` contains a non-ascii character.
Please change this.